lib/glific/third_party/bigquery/bigquery_worker.ex

```diff
@@ -391,6 +393,40 @@ defmodule Glific.BigQuery.BigQueryWorker do
     :ok
   end
 
+  defp queue_table_data("wa_reactions", organization_id, attrs) do
+    Logger.info(
+      "fetching wa_reactions data for org_id: #{organization_id} to send on bigquery with attrs: #{inspect(attrs)}"
+    )
+
+    get_query("wa_reactions", organization_id, attrs)
+    |> Repo.all()
+    |> Enum.reduce(
+      [],
+      fn row, acc ->
+        [
+          %{
+            id: row.id,
+            phone: row.contact.phone,
+            group_id: row.wa_group.id,
+            group_label: row.wa_group.label,
+            reaction: row.wa_reaction.reaction,
+            wa_message_id: row.wa_messages.id,
+            wa_message: row.wa_messages.body,
```

Comment:
It should be `wa_message_id` rather than `wa_messages_id`.

Also we don't need `wa_message` here.